tools/sggolangcilint/tools.go

```diff
@@ -32,6 +35,10 @@ func defaultConfigPath() string {
 	return sg.FromToolsDir(name, ".golangci.yml")
 }
 
+func defaultFixConfigPath() string {
+	return sg.FromToolsDir(name, ".golangci.fix.yml")
```

Comment:
In the future, perhaps we want two separate yaml files; one that runs linters with comes with auto-fix capabilities (modifies files in-place), and one that just runs checks.

Nit: maybe refer to this file as `.golangci.auto-fix.yml`?

Reply:
Actually, scratch this on isolating linters into _either_ `.golangci.fix.yml` or `.golangci.check.yml` files.

In the editor (Neovim), I want to be able to point my linter to look at ONE file (I don't think I can tell it to use two files) and then I want _all_ linters in that file.